hdr.go

```diff
@@ -455,6 +463,10 @@ func (h *Histogram) countsIndexFor(v int64) int {
 	return int(h.countsIndex(bucketIdx, subBucketIdx))
 }
 
+func (h *Histogram) getIntegerToDoubleValueConversionRatio() int64 {
```

Comment:
this field must be encoded as float64, 
see https://github.com/HdrHistogram/HdrHistogram/blob/e9e3d3ea7bfbc628863c7d20fd96489813690d84/src/main/java/org/HdrHistogram/AbstractHistogram.java#L1998

Reply:
with this change I can at least decode the resulting encoded string of an empty histogram:
`func TestEmpty(t *testing.T) {
	var lowest int64 = 1
	var highest int64 = 3600 * 1000 * 1000
	sigfigs := 3
	h := hdrhistogram.New(lowest, highest, sigfigs)
	bufferOut, _ := h.Encode(hdrhistogram.V2CompressedEncodingCookieBase)
	fmt.Printf("Empty histoblob: %s\n", string(bufferOut))
}`
yields:
Empty histoblob: HISTFAAAACR42pJpmSzMwMDACMXMDBAAYjNcm7yEwf4DVAQQAAD//1WhBIk=

which I can decode with python and I can verify it is equal to the same empty histogram.

Reply:
Thank you for finding this issue @ahothan. Fixed in the latest commit.